### ISSUE TITLE
Change some PV names.

### ISSUE THREAD

**Issue by fernandohds564** [lnls-dig/bpm-epics-ioc#37]
Follow some suggestions for the PV names.
 - `SamplesPre-(SP/RB) --> NrSamplesPre-(SP/RB)`
 - `SamplesPost-(SP/RB) --> NrSamplesPost-(SP/RB)`
 - `Shots-(SP/RB) --> NrShots-(SP/RB)`
 - `TriggerEvent-(Sel/Sts) --> Ctrl-(Sel/Sts)`: It seems more intuitive...
 - `Trigger-(Sel/Sts) --> TriggerType-(Sel/Sts)`: More descritive
 -  `DataTrigChan-(Sel/Sts)  --> TriggerDataChan-(Sel/Sts)`: I think this last name follow the same pattern of the other trigger configuration PVs.
 - `Status-Sts --> Status-Mon`: Since this PV has no selection associated with it, it seems more appropriate to use `-Mon`
 - `TriggerDataSel-(SP/RB) --> TriggerDataSel-(Sel/Sts)`: Change this PV to a enum type, with possible values given by `('A', 'B', 'C', 'D', 'X', 'Y', 'Sum', 'Q')` The default value coud be `'A'`.
 - Would it be possible to change the names:
        `(monit1, fofb, tbt, adc, adcswap, fofbpha, tbtpha) --> `
        `(Monit1, FOFB, TbT, ADC, ADCSwap, FOFBPha, TbTPha)`
 - Would it be possible to exclude the spaces in the names:
         `"External Trig" --> "ExternalTrig"`
         `"Data Trig" --> "DataTrig"`
         `"Software Trig" --> "SoftwareTrig"`
         `"Too Many Samples" --> "TooManySamples"`
         `"Too Few Samples" --> "TooFewSamples"`
         `"No Memory" --> "NoMemory"`
 - Would it be possible to use CamelCase for Enum strings?:
         `start --> Start`
         `normal --> Normal`
         `...`

**Comment by lerwys** [comment]
All changes implemented in range of commits from bbb6fea to 93988cc with the exception of:

1.  Would it be possible to exclude the spaces in the names:
```
"External` Trig" --> "ExternalTrig"
"Data Trig" --> "DataTrig"
"Software Trig" --> "SoftwareTrig"
"Too Many Samples" --> "TooManySamples"
"Too Few Samples" --> "TooFewSamples"
"No Memory" --> "NoMemory"
```

This is thought as a message to the operator, so spaces are easier to read. If you need to use the value for something the integer equivalent is always available. Does it make sense or were you thinking on something else when you suggested the message without spaces?

**Comment by fernandohds564** [comment]
It does make sense @lerwys, thanks for implementing the suggestions!
I will close this issue.
